ext/json/json_arginfo.h

```diff
@@ -0,0 +1,20 @@
+/* This is a generated file, edit the .stub.php file instead. */
```

Comment:
@nikic would it make sense to extend the generator so it mention in this comment with which command this file can be generated?
(Maybe also add a version/date stamp of the generator used?)

Reply:
Sounds reasonable (the first part).